lib/proxi/local/sdrf_handler.py

```
@@ -0,0 +1,729 @@
+                        elif title == 'source name' and preserve_main_source_name:
+                            value = main_value
+                            if tag_with_decisions:
+                                value += "_SpecialRuleKeepMain"
+                            self.log_problem('WARNING', 'SpecialRuleKeepMain', irow + 1, icolumn + 1, f"For column '{title}', values in main ({main_value}) and merge ({merge_value}) differ. A special rule always keeps the main.")
+                        else:
+                            value = merge_value
+                            if tag_with_decisions:
+                                value += "_preferMergeOverMain"
+                            self.log_problem('WARNING', 'MergeValueOverridesMain', irow + 1, icolumn + 1, f"For column '{title}', values in main ({main_value}) and merge ({merge_value}) differ. Value from merge ({merge_value}) supercedes.")
+                elif len(merged_column['main_icolumn']) > 1:
+                    for imulti_column in merged_column['main_icolumn']:
```

Ruff (0.15.12)

[warning] Loop control variable `imulti_column` not used within loop body

Rename unused `imulti_column` to `_imulti_column`

(B007)